BUILD

```diff
@@ -69,3 +84,12 @@ filegroup(
         "//rules:bins",
     ],
 )
+
+# Below this line is for development purposes only and should thus not be
```

Comment:
But we have no way to strip it out right now.  Can you get this out of this BUILD and put it in a different one.